### PR TITLE
Reimplemented admin room pages

### PR DESCRIPTION
**Pull request by MyIgel**
* Reimplemented admin room pages
* Unified macros to contain more options
* Updated session form data handling to be actually useable

### CODE REVIEW

File: resources/views/macros/form.twig [hunk 1]
```diff
@@ -1,5 +1,10 @@
 {% macro entry_required() %}
-    <span class="bi bi-exclamation-triangle text-info"></span>
+    <span class="text-info" title="{{ __('form.required') }}">*</span>
+{%- endmacro %}
+
+{% macro info(text) %}
+    <span class="bi bi-info-circle-fill text-info" data-bs-toggle="tooltip"
+        data-bs-title="{{ text | e('html_attr') }}"></span>
```
Comment (weeman1337) [inline resources/views/macros/form.twig:7]:
```suggestion
        title="{{ text | e('html_attr') }}"></span>
```

https://getbootstrap.com/docs/5.3/components/tooltips/

image

File: resources/views/admin/rooms/edit.twig [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+{% block title %}{{ room ? __('room.edit.title') : __('room.create.title') }}{% endblock %}
+
+{% block row_content %}
+    <form method="post">
+        {{ csrf() }}
+        {{ f.hidden('id', room ? room.id : '') }}
+
+        <div class="row">
+            <div class="col-lg-6">
+                {{ f.input('name', __('room.name'), null, {'required': true, 'entry_required_icon': true, 'value': f.formData('room', room ? room.name : '')}) }}
```
Comment (weeman1337) [inline resources/views/admin/rooms/edit.twig:14]: Can you split the long lines?

File: src/Controllers/Admin/RoomsController.php [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+            $request,
+            [
+                'name'        => 'required',
+                'description' => 'required|optional',
+                'dect'        => 'required|optional',
+                'map_url'     => 'optional|url',
+            ] + $validation
+        );
+
+        if (Room::whereName($data['name'])->where('id', '!=', $room->id)->count()) {
```
Comment (weeman1337) [inline src/Controllers/Admin/RoomsController.php:85]:
```suggestion
        if (Room::whereName($data['name'])->where('id', '!=', $room->id)->exists()) {
```

File: resources/assets/js/forms.js [hunk 4]
```diff
@@ -254,6 +254,13 @@ ready(() => {
   document.querySelectorAll('[data-bs-toggle="popover"]').forEach((element) => new bootstrap.Popover(element));
 });
 
+/**
+ * Init Bootstrap Tooltips
+ */
+ready(() => {
+  document.querySelectorAll('[data-bs-toggle="tooltip"]').forEach((element) => new bootstrap.Tooltip(element));
```
Comment (ximex) [inline resources/assets/js/forms.js:261]: i would merge that with the popover init above

Comment (MyIgel) [reply resources/assets/js/forms.js:261]: @weeman1337 wants to do that later for the whole file

**Pull request merged**